### ISSUE TITLE
Keep bundle.js under 500Kb unminified.

### ISSUE THREAD

**Issue by mbebenita** [wasdk/WebAssemblyStudio#52]
The current size is 1.18 MB, webpack-bundle-size-analyzer prints out:

```
pako: 213.83 KB (18.9%)
wasmparser: 160.01 KB (14.2%)
jszip: 140.69 KB (12.4%)
  core-js: 9.31 KB (6.62%)
  <self>: 131.38 KB (93.4%)
readable-stream: 66.22 KB (5.86%)
buffer: 47.47 KB (4.20%)
cassowary: 47.01 KB (4.16%)
react-modal: 33.15 KB (2.93%)
mousetrap: 32.28 KB (2.85%)
prop-types: 24.88 KB (2.20%)
minimatch: 24.75 KB (2.19%)
util: 16.05 KB (1.42%)
  inherits: 672 B (4.09%)
  <self>: 15.4 KB (95.9%)
events: 8.13 KB (0.719%)
string_decoder: 8.1 KB (0.717%)
setimmediate: 6.32 KB (0.559%)
path-browserify: 6.04 KB (0.534%)
lie: 5.79 KB (0.512%)
process: 5.29 KB (0.468%)
brace-expansion: 4.68 KB (0.414%)
fbjs: 4.26 KB (0.377%)
stream-browserify: 3.54 KB (0.313%)
base64-js: 3.39 KB (0.300%)
core-util-is: 2.95 KB (0.261%)
object-assign: 2.06 KB (0.182%)
ieee754: 2.01 KB (0.177%)
immediate: 1.84 KB (0.163%)
warning: 1.76 KB (0.156%)
util-deprecate: 1.58 KB (0.139%)
safe-buffer: 1.49 KB (0.132%)
timers-browserify: 1.33 KB (0.117%)
balanced-match: 1.15 KB (0.101%)
process-nextick-args: 1.02 KB (0.0898%)
webpack: 1 KB (0.0886%)
exenv: 863 B (0.0745%)
inherits: 672 B (0.0580%)
concat-map: 345 B (0.0298%)
isarray: 132 B (0.0114%)
<self>: 248.61 KB (22.0%)
```

**Comment by mbebenita** [comment]
@haxxorsid jszip is too large to be included in `bundle.js`, especially since it's not used until downloading is requested. Can you move it into a dynamically loaded module? Or work out a simpler way to package files for downloading.

**Comment by yurydelendik** [comment]
> Can you move it into a dynamically loaded module?

webpack has [code splitting](https://webpack.js.org/guides/code-splitting/) future which needs to be used here

**Comment by yurydelendik** [comment]
To make this issue more specific:
- [ ] add a test for 500k-limit for the main bundle

**Comment by yurydelendik** [comment]
Fixed by #133
